week-1-practise/level-1/7-logic-error-4.js

```diff
@@ -1,7 +1,7 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function multiply(a, b, c) {
-  a * b * c;
+  return a * b * c;
   return;
```

Comment:
There's a line of code here that will never be executed. Can you spot it?

Reply:
thanks, of course `return;`